packages/zero-schema/src/builder/table-builder.ts

```diff
@@ -43,6 +43,22 @@ export function json<T extends ReadonlyJSONValue = ReadonlyJSONValue>() {
   });
 }
 
+export function date<T extends number = number>() {
```

Comment:
We could get away with just a 'timestamp' time that could cover all the Date/Time Types (https://www.postgresql.org/docs/current/datatype-datetime.html), including date, time, timestamp, timestamptz.  We don't currently support time.  

I'm torn... I don't think of a date as a timestamp, so it feels a bit more intuitive to have a separate date type.   Also time is even less like a timestamp (though you'd represent it as millis from midnight).